bl-service/src/main/java/com/ayeeti/blservice/user/requests/CreateUserAtLocation.java

```diff
@@ -0,0 +1,21 @@
+package com.ayeeti.blservice.user.requests;
+
+import com.ayeeti.blservice.location.requests.LocationRequest;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Set;
+
+@Builder
+@AllArgsConstructor
+@Getter
+@Setter
+public class CreateUserAtLocation {
```

Comment:
As any request this should have a request in it's name. e.g CreateUserAtLocationRequest and it should be a record! not a class

Reply:
I was having hard time with intellisense with the record so I changed to the class.